src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -135,6 +135,7 @@ public net.minecraft.client.renderer.texture.SpriteContents f_243731_ # byMipLev
 default net.minecraft.client.renderer.texture.SpriteContents f_244575_ # animatedTexture
 default net.minecraft.client.renderer.texture.SpriteContents m_245088_()I # getFrameCount
 public net.minecraft.client.resources.ClientPackSource m_246691_(Ljava/nio/file/Path;)Lnet/minecraft/server/packs/VanillaPackResources; # createVanillaPackSource
+protected net.minecraft.client.resources.TextureAtlasHolder f_118884_ # textureAtlas
```

Comment:
Why `protected` and not `public`? _What is the usecase?_

Reply:
my usecase is custom atlases that previously had to provide the list of resource locations to load and now have no idea what is in the atlas after loading (which is data I need)
the only vanilla atlases that use this MobEffectTextureManager and PaintingTextureManager
if you think public would be better then I'll change that